### PR TITLE
Corrections and additions

### LINKED ISSUE #427: OpenRCT2
For OpenRCT2 the repo name should really be https://github.com/OpenRCT2/OpenRCT2

Requires an edit to this line.
https://github.com/piranha/osgameclones/blob/master/games/r.yaml#L216

It is also compatible with most of RCT1 so I guess it should really be under both categories.

### PR DESCRIPTION
**Pull request by wonea**
Fixes #427 #424 #435

### CODE REVIEW

File: games/d.yaml [hunk 1]
```diff
@@ -393,6 +393,11 @@
     development: active
     lang: C++
     license: [GPL3]
+  - AdaDoom3
+    updated: 2017-04-04
+    lang: Ada
+    repo: https://github.com/AdaDoom3/AdaDoom3
+    development: active
```
Comment (cxong) [inline games/d.yaml]: This project's last commit was February; it would be better to call it sporadic

File: games/s.yaml [hunk 2]
```diff
@@ -402,6 +402,19 @@
     - http://ultrastardx.sourceforge.net/screenshots/game_visualisations.png
     - http://ultrastardx.sourceforge.net/screenshots/partymode_teams.png
 
+- name: [Snipes, Snipes (video game)]
+  remakes:
```
Comment (cxong) [inline games/s.yaml]: This needs a name and yaml list member i.e.

    - name: Snipes
      remakes:
      - name: Snipes
        repo: ...

Comment (wonea) [inline games/s.yaml]: Sorted

**Comment by wonea** [comment]
@cxong are you happy with the other additions?

**Pull request merged**